examples/circuitplayground_acceleration_mapping_neopixels.py

```diff
@@ -0,0 +1,52 @@
+"""Maps acceleration (tilt) to Neopixel colors.
+
+x, y, and z acceleration components map to red, green and blue,
+respectively.
+
+When the CPX is level, the lights are blue because there is no acceleration
+on x or y, but on z, gravity pulls at 9.81 meters per second per second (m/s²).
+When banking, the vertical (z) axis is no longer directly aligned with gravity,
+so the blue decreases, and red increases because gravity is now pulling more
+along the x axis. Similarly, when changing the pitch from level, we see blue change
+to green.
+
+This video walks you through the code: https://youtu.be/eNpPLbYx-iA
+"""
+
+from time import sleep
```

Comment:
We try to avoid doing `from foo import bar` as it isn't as memory efficient as importing the lib alone and using it when needed. Please change this to `import time` and update the related code.

Reply:
Interesting! Is this a CircuitPython peculiarity? I’d never heard of that.

Reply:
I'm not sure, to be honest. We did a series of tests importing only the lib, importing individual parts of the lib using `from`, and doing `from foo import *`, and found the memory efficiency was best with importing the lib alone and then using it as needed. Something to do with the last two importing things before they're needed and using more memory, if I remember correctly.